plugins/woocommerce/client/legacy/js/frontend/a8c-address-autocomplete-service.js

```
@@ -0,0 +1,459 @@
+						lang: document.documentElement.lang,
+						token: value.key,
+					} );
+
+					const response = await fetch(
+						`${ selectUrl }?${ params.toString() }`
+					);
+
+					let data = await response.json();
+					// Reset session ID after successful select
+					sessionId =
+						crypto && crypto.randomUUID
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)